### PR TITLE
Test deployments

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- Fix problem with circles being above modal
- Test deployments

**Related issue(s)**

### CODE REVIEW

**Review by Sawthis — APPROVED** [review]

File: tests/tests/busola-create-a-deployment.spec.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  });
+
+  it('Check if deployment and service exist', () => {
+    cy.getIframeBody()
+      .contains('a', DEPLOYMENT_NAME)
+      .should('be.visible');
+
+    getLeftNav()
+      .contains('Discovery and Network')
+      .click({ force: true });
```
Comment (parostatkiem) [inline tests/tests/busola-create-a-deployment.spec.js:151]: Is there any reason you use `force` here and below?

File: tests/tests/busola-create-a-deployment.spec.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+    getLeftNav()
+      .contains('Services')
+      .click({ force: true })
+      .wait(2000);
```
Comment (parostatkiem) [inline tests/tests/busola-create-a-deployment.spec.js:156]: You could use `get` or `find` below instead. Cypress will retry looking for the desired element.

File: tests/cypress.json [hunk 3]
```diff
@@ -3,9 +3,11 @@
   "video": false,
   "defaultCommandTimeout": 5000,
   "responseTimeout": 4000,
-
   "fixturesFolder": "fixtures",
   "integrationFolder": "tests",
+  "pageLoadTimeout": 10000,
```
Comment (parostatkiem) [inline tests/cypress.json:8]: I think we should adjust all the timeouts to our *expectations* to also test if the performance of Busola isn't below the expected threshold. A page loading for 9.5s isn't a success, don't you think?

**Review by parostatkiem — CHANGES_REQUESTED** [review]

**Review by Sawthis — CHANGES_REQUESTED** [review]
Please look on Jan's comments.

File: tests/tests/busola-create-a-deployment.spec.js [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+const NAMESPACE_NAME = `b-busola-test-${random}`;
+
+const DOCKER_IMAGE = 'eu.gcr.io/kyma-project/pr/orders-service:PR-162';
+const DEPLOYMENT_NAME = 'orders-service';
+
+context('Busola - Create a Deployment', () => {
+  const getLeftNav = () => cy.get('nav[data-testid=semiCollapsibleLeftNav]');
+
+  before(() => {
+    cy.visit(ADDRESS)
```
Comment (Sawthis) [inline tests/tests/busola-create-a-deployment.spec.js:19]: You need to adjust login flow. Example is here: https://github.com/kyma-project/busola/blob/main/tests/tests/busola-smoke-tests.spec.js#L18

**Review by Sawthis — CHANGES_REQUESTED** [review]

**Pull request merged**